libs/s25main/Window.h

```diff
@@ -98,8 +98,10 @@ class Window
     void LockRegion(Window* window, const Rect& rect);
     /// Gibt eine gesperrte Region wieder frei.
     void FreeRegion(Window* window);
-    /// Check if the gicen point is in a region locked by any window other than exception
+    /// Check if the given point is in a region locked by any window other than exception
     bool IsInLockedRegion(const Position& pos, const Window* exception = nullptr) const;
+    bool IsMouseOver() const;
```

Comment:
can you add a comment on what those two methods are for?

Reply:
Done. Also added a test for them